src/App.js

```diff
@@ -1,25 +1,55 @@
 import React from 'react';
-
+import { Button } from 'react-bootstrap';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import Clock from './components/Clock';
 import './App.scss';
 
-const App = () => {
-  setInterval(() => {
-    const date = new Date();
+class App extends React.Component {
+  state = {
+    isClockVisible: true,
+    clockName: 1,
+  }
+
+  visibleClock = () => {
+    this.setState({ isClockVisible: true });
+  }
+
+  hiddenClock = () => {
+    this.setState({ isClockVisible: false });
+  }
+
+  randomName = () => {
```

Comment:
```suggestion
  createRandomName = () => {
```
function name needs to be a verb